pandas/core/generic.py

```diff
@@ -5455,6 +5455,16 @@ def __finalize__(
                 object.__setattr__(self, name, getattr(other, name, None))
 
         if method == "concat":
+            # Issue #41828, retain the attrs only if all NDFrame have the same
```

Comment:
this is already handled above no? L5448

Reply:
@jreback Thanks for the review. No.
So during concatenation, the `other` is a `concatenator`, which is not an `NDFrame`.
So it will not pass the if statement in L5447.

Reply:
can remove the comment